framework/src/media/audio/audio_manager.h

```diff
@@ -376,6 +436,149 @@ audio_manager_result_t set_input_audio_gain(uint8_t gain);
  ****************************************************************************/
 audio_manager_result_t set_output_audio_volume(uint8_t volume);
 
+/****************************************************************************
+ * Name: foreach_supported_process_device
```

Comment:
This name should be find_stream_in_device_with_process_type.

Reply:
done